src/main/java/edu/oakland/production/database/DatabaseGisManager.java

```diff
@@ -6,8 +6,8 @@
 
   public void passStoreRequest(LocationDataPoint locationDataPoint);
   
-  public void passModeRequest(String mode);
+  public String passModeRequest(String mode);
```

Comment:
Generally setter should be void return because you know what you set. Also, we want to see if we can avoid changing the public interface unnecessarily, as that could break other code from Middleware.
```suggestion
  public void passModeRequest(String mode);
```